README.md

```diff
@@ -15,67 +17,21 @@ For each of the three exchanges `USD`, `EUR` and `JPY`, this demo makes use of t
 - `CrytoTraderQuotes{USD/EUR/JPY}`
 - `CryptoTraderQuotesAvg{USD/EUR/JPY}`
 
-The below steps will describe on how to deploy the node and UI application.
+The below steps will describe on how set up the frontend.
 
-# 2. Prerequisites
+# 1. Prerequisites
```

Comment:
```suggestion
# 1. Clone or Fork the Repo
```

Reply:
Prerequisites are things you need to do before you do the procedure, they are never numbered steps. Looks like this is step 1 and you need a Prerequisites section before it, which would include having nodejs and npm installed as well as completing the backend Crypto Trading App.